[2022-11-16T09:20:42.284+0000] {processor.py:154} INFO - Started process (PID=32) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:20:42.303+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:20:42.307+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:20:42.306+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:20:42.396+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:20:42.383+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 21, in get_yellow_brick_version
    import yellowbrick
ModuleNotFoundError: No module named 'yellowbrick'
[2022-11-16T09:20:42.399+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:20:42.496+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 0.230 seconds
[2022-11-16T09:21:12.573+0000] {processor.py:154} INFO - Started process (PID=33) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:21:12.577+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:21:12.579+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:21:12.578+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:21:12.613+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:21:12.609+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 21, in get_yellow_brick_version
    import yellowbrick
ModuleNotFoundError: No module named 'yellowbrick'
[2022-11-16T09:21:12.615+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:21:12.640+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 0.072 seconds
[2022-11-16T09:21:42.807+0000] {processor.py:154} INFO - Started process (PID=34) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:21:42.811+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:21:42.813+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:21:42.813+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:21:42.847+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:21:42.843+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 21, in get_yellow_brick_version
    import yellowbrick
ModuleNotFoundError: No module named 'yellowbrick'
[2022-11-16T09:21:42.850+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:21:42.876+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 0.075 seconds
[2022-11-16T09:22:13.043+0000] {processor.py:154} INFO - Started process (PID=35) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:22:13.046+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:22:13.048+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:22:13.048+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:22:13.075+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:22:13.072+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 21, in get_yellow_brick_version
    import yellowbrick
ModuleNotFoundError: No module named 'yellowbrick'
[2022-11-16T09:22:13.077+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:22:13.101+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 0.063 seconds
[2022-11-16T09:22:35.439+0000] {processor.py:154} INFO - Started process (PID=33) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:22:35.442+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:22:35.444+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:22:35.444+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:22:35.692+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:22:35.692+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-33zvyvao because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:22:35.907+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:22:35.906+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:22:37.211+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:22:37.217+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:22:37.214+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:22:37.220+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:22:37.241+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 1.805 seconds
[2022-11-16T09:23:07.527+0000] {processor.py:154} INFO - Started process (PID=36) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:23:07.530+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:23:07.532+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:23:07.531+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:23:07.740+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:23:07.740+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-gqnh7vt5 because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:23:07.926+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:23:07.926+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:23:08.970+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:23:08.975+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:23:08.972+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:23:08.977+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:23:08.997+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 1.474 seconds
[2022-11-16T09:23:39.263+0000] {processor.py:154} INFO - Started process (PID=39) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:23:39.266+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:23:39.268+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:23:39.268+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:23:39.459+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:23:39.459+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-yi9b32n6 because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:23:39.623+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:23:39.623+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:23:40.744+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:23:40.755+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:23:40.748+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:23:40.757+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:23:40.800+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 1.542 seconds
[2022-11-16T09:24:11.128+0000] {processor.py:154} INFO - Started process (PID=42) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:24:11.131+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:24:11.133+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:24:11.133+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:24:11.313+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:24:11.313+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-megy8vnv because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:24:11.481+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:24:11.481+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:24:12.776+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:24:12.783+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:24:12.779+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:24:12.785+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:24:12.808+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 1.684 seconds
[2022-11-16T09:24:43.095+0000] {processor.py:154} INFO - Started process (PID=45) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:24:43.098+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:24:43.099+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:24:43.099+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:24:43.281+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:24:43.281+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-th1r2i09 because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:24:43.448+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:24:43.448+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:24:44.355+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:24:44.362+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:24:44.358+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:24:44.364+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:24:44.386+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 1.295 seconds
[2022-11-16T09:25:14.639+0000] {processor.py:154} INFO - Started process (PID=48) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:25:14.642+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:25:14.643+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:25:14.643+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:25:14.845+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:25:14.845+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-dec7t3hj because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:25:15.020+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:25:15.020+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:25:16.040+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:25:16.047+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:25:16.043+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:25:16.050+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:25:16.075+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 1.441 seconds
[2022-11-16T09:25:46.340+0000] {processor.py:154} INFO - Started process (PID=51) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:25:46.344+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:25:46.346+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:25:46.345+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:25:46.528+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:25:46.528+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-3k93s5tk because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:25:46.701+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:25:46.701+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:25:47.576+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:25:47.581+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:25:47.578+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:25:47.583+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:25:47.605+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 1.269 seconds
[2022-11-16T09:26:17.922+0000] {processor.py:154} INFO - Started process (PID=54) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:26:17.926+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:26:17.929+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:26:17.929+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:26:18.099+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:26:18.099+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-hg7kl13y because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:26:18.263+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:26:18.263+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:26:19.362+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:26:19.370+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:26:19.365+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:26:19.373+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:26:19.396+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 1.479 seconds
[2022-11-16T09:26:49.691+0000] {processor.py:154} INFO - Started process (PID=57) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:26:49.693+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:26:49.695+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:26:49.695+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:26:49.942+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:26:49.942+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-652_s7vz because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:26:50.174+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:26:50.174+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:26:51.634+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:26:51.647+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:26:51.640+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:26:51.653+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:26:51.696+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 2.011 seconds
[2022-11-16T09:27:21.889+0000] {processor.py:154} INFO - Started process (PID=60) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:27:21.891+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:27:21.892+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:27:21.892+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:27:22.088+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:27:22.088+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-a3348d4o because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:27:22.250+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:27:22.249+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:27:23.200+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:27:23.206+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:27:23.203+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:27:23.208+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:27:23.226+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 1.341 seconds
[2022-11-16T09:27:53.482+0000] {processor.py:154} INFO - Started process (PID=63) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:27:53.486+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:27:53.488+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:27:53.488+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:27:53.774+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:27:53.774+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-l4iqlvvk because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:27:54.084+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:27:54.083+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:27:55.392+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:27:55.401+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:27:55.395+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:27:55.406+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:27:55.427+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 1.950 seconds
[2022-11-16T09:28:25.491+0000] {processor.py:154} INFO - Started process (PID=66) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:28:25.494+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:28:25.496+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:28:25.496+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:28:25.684+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:28:25.684+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-v0r6lurf because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:28:25.845+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:28:25.844+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:28:26.725+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:28:26.737+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:28:26.733+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:28:26.739+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:28:26.760+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 1.273 seconds
[2022-11-16T09:28:56.917+0000] {processor.py:154} INFO - Started process (PID=69) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:28:56.919+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:28:56.921+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:28:56.921+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:28:57.102+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:28:57.102+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-gxo2_huy because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:28:57.309+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:28:57.308+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:28:58.822+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:28:58.833+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:28:58.827+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:28:58.835+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:28:58.859+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 1.948 seconds
[2022-11-16T09:29:29.157+0000] {processor.py:154} INFO - Started process (PID=72) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:29:29.161+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:29:29.163+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:29:29.163+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:29:29.348+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:29:29.348+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-6daf990j because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:29:29.509+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:29:29.508+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:29:30.367+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:29:30.373+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:29:30.370+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:29:30.376+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:29:30.398+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 1.245 seconds
[2022-11-16T09:32:50.730+0000] {processor.py:154} INFO - Started process (PID=33) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:32:50.741+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:32:50.753+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:32:50.751+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:32:51.387+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:32:51.387+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-a8u90yd3 because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:32:51.986+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:32:51.986+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:32:54.724+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:32:54.749+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:32:54.738+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 51, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:32:54.752+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:32:54.899+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 4.199 seconds
[2022-11-16T09:33:10.575+0000] {processor.py:154} INFO - Started process (PID=36) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:33:10.586+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:33:10.598+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:33:10.598+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:33:12.646+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:33:12.645+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-6erg2qw8 because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:33:13.508+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:33:13.503+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:33:17.560+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:33:17.600+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:33:17.570+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 52, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:33:17.605+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:33:17.687+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 7.139 seconds
[2022-11-16T09:33:48.262+0000] {processor.py:154} INFO - Started process (PID=39) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:33:48.265+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:33:48.267+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:33:48.267+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:33:48.538+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:33:48.538+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-45rwi73a because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-16T09:33:48.754+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:33:48.754+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-16T09:33:49.981+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-16T09:33:49.995+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:33:49.991+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/dag_test_with_python_dependence.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_test_with_python_dependence.py", line 52, in <module>
    python_callable=get_yellow_brick_version()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 408, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 162, in __init__
    raise AirflowException('`python_callable` param must be callable')
airflow.exceptions.AirflowException: `python_callable` param must be callable
[2022-11-16T09:33:50.002+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:33:50.048+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 1.790 seconds
[2022-11-16T09:34:01.402+0000] {processor.py:154} INFO - Started process (PID=42) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:34:01.408+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:34:01.411+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:34:01.410+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:34:01.518+0000] {processor.py:766} INFO - DAG(s) dict_keys(['python_dependences']) retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:34:02.711+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:34:02.710+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2022-11-16T09:34:02.816+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:34:02.816+0000] {dag.py:3340} INFO - Setting next_dagrun for python_dependences to 2022-11-16T00:00:00+00:00, run_after=2022-11-17T00:00:00+00:00
[2022-11-16T09:34:02.863+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 1.469 seconds
[2022-11-16T09:34:33.239+0000] {processor.py:154} INFO - Started process (PID=43) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:34:33.243+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:34:33.245+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:34:33.245+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:34:33.304+0000] {processor.py:766} INFO - DAG(s) dict_keys(['python_dependences']) retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:34:33.382+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:34:33.382+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2022-11-16T09:34:33.420+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:34:33.420+0000] {dag.py:3340} INFO - Setting next_dagrun for python_dependences to 2022-11-16T00:00:00+00:00, run_after=2022-11-17T00:00:00+00:00
[2022-11-16T09:34:33.442+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 0.210 seconds
[2022-11-16T09:34:57.573+0000] {processor.py:154} INFO - Started process (PID=44) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:34:57.582+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:34:57.592+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:34:57.591+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:34:57.736+0000] {processor.py:766} INFO - DAG(s) dict_keys(['python_dependences']) retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:34:57.829+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:34:57.829+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2022-11-16T09:34:57.861+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:34:57.861+0000] {dag.py:3340} INFO - Setting next_dagrun for python_dependences to 2022-11-16T00:00:00+00:00, run_after=2022-11-17T00:00:00+00:00
[2022-11-16T09:34:57.892+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 0.341 seconds
[2022-11-16T09:35:28.028+0000] {processor.py:154} INFO - Started process (PID=45) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:35:28.032+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:35:28.034+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:35:28.034+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:35:28.068+0000] {processor.py:766} INFO - DAG(s) dict_keys(['python_dependences']) retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:35:28.121+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:35:28.121+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2022-11-16T09:35:28.154+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:35:28.154+0000] {dag.py:3340} INFO - Setting next_dagrun for python_dependences to 2022-11-16T00:00:00+00:00, run_after=2022-11-17T00:00:00+00:00
[2022-11-16T09:35:28.173+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 0.151 seconds
[2022-11-16T09:35:58.591+0000] {processor.py:154} INFO - Started process (PID=46) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:35:58.594+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:35:58.596+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:35:58.596+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:35:58.641+0000] {processor.py:766} INFO - DAG(s) dict_keys(['python_dependences']) retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:35:58.730+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:35:58.730+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2022-11-16T09:35:58.769+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:35:58.769+0000] {dag.py:3340} INFO - Setting next_dagrun for python_dependences to 2022-11-16T00:00:00+00:00, run_after=2022-11-17T00:00:00+00:00
[2022-11-16T09:35:58.789+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 0.202 seconds
[2022-11-16T09:48:23.923+0000] {processor.py:154} INFO - Started process (PID=67) to work on /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:48:23.926+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/dag_test_with_python_dependence.py for tasks to queue
[2022-11-16T09:48:23.927+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:48:23.927+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:48:23.988+0000] {processor.py:766} INFO - DAG(s) dict_keys(['python_dependences']) retrieved from /opt/airflow/dags/dag_test_with_python_dependence.py
[2022-11-16T09:48:24.090+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:48:24.090+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2022-11-16T09:48:24.118+0000] {logging_mixin.py:120} INFO - [2022-11-16T09:48:24.118+0000] {dag.py:3340} INFO - Setting next_dagrun for python_dependences to 2022-11-16T00:00:00+00:00, run_after=2022-11-17T00:00:00+00:00
[2022-11-16T09:48:24.147+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/dag_test_with_python_dependence.py took 0.231 seconds
